wazimap_np/health.py

```diff
@@ -71,6 +95,12 @@ def get_health_profile(geo_code, geo_level, session):
         child_nourishment, _ = child_nourishment_table.get_stat_data(
             geo_level, geo_code, percent=False)
 
+        health_facilities_data, _ = get_stat_data(
+            ['facilitytype'], geo_level, geo_code, session,
+            recode=dict(HEALTH_CENTER_RECODES),
+            key_order=HEALTH_CENTER_RECODES.values(),
```

Comment:
If you use `order_by='-total'` instead of `key_order=HEALTH_CENTER_RECODES.values()`, then the bars will go from largest to smallest every time, like this:

![dadeldhura-greatest-to-least](https://user-images.githubusercontent.com/3824492/32203463-6b66817c-bdb1-11e7-8e71-050bce2c918a.png)

Would you consider updating this pull request or making another pull request to make this change?

Reply:
Sure, I'll update this pr.